.github/workflows/source-maintenance.yaml

```
@@ -0,0 +1,50 @@
+              --no-cache-dir \
+              --disable-pip-version-check \
+              --quiet \
+              --progress-bar off \
+              codespell==1.16 # TODO: Upgrade to codespell v2
+
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: ${{ env.CHECKOUT_FETCH_DEPTH }}
+
+      - run: |
+        ./scripts/source-maintenance.sh --check-and-update-copyright no --only-changed-since fork --update-contributors-since never
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 41: could not find expected ':'

(syntax-check)